packages/react-card/src/stories/CardScale.stories.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+
+import { Body, Caption } from '@fluentui/react-text';
+
+import { Button } from '@fluentui/react-button';
+import { MoreVerticalRegular } from '@fluentui/react-icons';
+import { Card, CardHeader } from '../index'; // codesandbox-dependency: @fluentui/react-card ^9.0.0-beta
```

Comment:
```suggestion
import { Card, CardHeader } from '../index';
```

Thankfully this is no longer needed after microsoft/fluentui-storybook-addons#9